lib/landrush.rb

```diff
@@ -12,6 +14,10 @@ def self.working_dir
   def self.working_dir=(working_dir)
     @working_dir = Pathname(working_dir).tap(&:mkpath)
   end
+
+  def self.config
+    @config ||= PStore.new(working_dir.join('config.pstore'))
```

Comment:
That means the config is not a pstore which is not human readable anymore, right? I think this is a bit unfortunate. Having the ability to easily view/edit the configuration is imo a nice thing.